presto-cassandra/src/test/java/io/prestosql/plugin/cassandra/CassandraQueryRunner.java

```diff
@@ -44,16 +44,16 @@ public static synchronized DistributedQueryRunner createCassandraQueryRunner()
 
         queryRunner.installPlugin(new CassandraPlugin());
         queryRunner.createCatalog("cassandra", "cassandra", ImmutableMap.of(
-                "cassandra.contact-points", EmbeddedCassandra.getHost(),
-                "cassandra.native-protocol-port", Integer.toString(EmbeddedCassandra.getPort()),
+                "cassandra.contact-points", CassandraServer.getHost(),
```

Comment:
i am thinking it could make sense to convert CassandraServer to normal object, rather than having it a static instance.

-- are we benefiting from the reuse of single cassandra instance across test classes?

FWIW cassandra module runs tests single threaded today (https://github.com/prestosql/presto/blob/7aaa0f5fac1454b23e4bd2ac3a511d749bd00fbe/presto-cassandra/pom.xml#L17)

* this means we don't need to worry about increased mem usage from multiple tests runs now
* having CassandraServer as an object could allow us to run the tests in 2 threads (memory permitting)

Reply:
how long does it take to start cassandra?

Reply:
It takes about:
* 37s when the image does not exist in the environment
* 17s when the image already exist in the environment

Reply:
Making CassandraServer non-static with 2 threads will extend the test time for a few minutes. 
14m 34s → 16m 58s

Reply:
I think it's worth it, since it brings more isolation. (Even if we don't get the additional benefit of parallel test execution, which could bring the execution time back lower).

17 minutes is still a decent run time for a test configuration.

(BTW Are there any test classes that could be easily merged?)

Reply:
I think so too. Let me push the additional commit. 

Currently, dockerized CassandraServer is called from 4 places.
1. TestCassandraDistributedQueries
2. TestCassandraIntegrationSmokeTest
3. TestCassandraConnector
4. TestCassandraTokenSplitManager

We could merge 3 & 4 technically, but I would keep the current implementation.